adapters/beachfront/beachfront.go

```diff
@@ -435,43 +480,49 @@ func (a *BeachfrontAdapter) MakeBids(internalRequest *openrtb.BidRequest, extern
 	for i := 0; i < len(bids); i++ {
 		bidResponse.Bids = append(bidResponse.Bids, &adapters.TypedBid{
 			Bid:     &bids[i],
-			BidType: bidtype,
+			BidType: getBidType(bids[i]),
 		})
 	}
 
 	return bidResponse, errs
 }
 
-func postprocess(response *adapters.ResponseData, externalRequest *adapters.RequestData, id string, bidtype openrtb_ext.BidType) ([]openrtb.Bid, []error) {
-	var beachfrontResp []BeachfrontResponseSlot
+func postprocess(response *adapters.ResponseData, externalRequest *adapters.RequestData, id string) ([]openrtb.Bid, []error) {
+	var beachfrontResp []beachfrontResponseSlot
 	var errs = make([]error, 0)
+	// var list = json.Unmarshal()
```

Comment:
Just a nit pick but since there are other changes can you please remove this comment?

Reply:
Thank you for nit picking. Done locally.